cudapoa/src/cudapoa_structs.cuh

```diff
@@ -37,6 +37,10 @@
 // ad-hoc maximum band-width defined in adaptive banding
 #define CUDAPOA_MAX_ADAPTIVE_BAND_WIDTH 1536
 
+// rerun codes used in adaptive banding
+#define SHIFT_ADAPTIVE_BAND_TO_LEFT -10
```

Comment:
also prefix these macros with `CUDAPOA_KERNEL`

Reply:
this convention did not seem to be applied in all cases, for example `CELLS_PER_THREAD`, therefore a couple of macros were left unchanged. Added `CUDAPOA` to all, including `CELLS_PER_THREAD`

Reply:
ah got it, good point. thanks for fixing the rest!